client/src/app/pages/assessment-management/assessment-settings/assessment-settings-page.tsx

```diff
@@ -274,6 +275,14 @@ const AssessmentSettings: React.FC = () => {
                             />
                           </Td>
                           <Td width={25} {...getTdProps({ columnKey: "name" })}>
+                            {questionnaire.system && (
+                              <LockIcon
+                                style={{
+                                  marginRight:
+                                    "var(--pf-v5-global--spacer--sm)",
+                                }}
```

Comment:
Looks like there is a different way to do inline icons (at least with PF5) that don't involve touching the icon style https://www.patternfly.org/components/icon#inline

Reply:
I'm a bit confused about you're suggestion, all I meant here is to give a proper spacing from the icon to the text so they won't be too close.
I tried to do as suggested in the example to something like, which didn't gave me the margin I was looking for:
```
<Td width={25} {...getTdProps({ columnKey: "name" })}>
  <TextContent>
    <Text component="p">
      {questionnaire.system && (
        <Icon isInline>
          <LockIcon />
        </Icon>
      )}
      {questionnaire.name}
    </Text>
  </TextContent>
</Td>
```

Reply:
Also, you can use the "spacing" utility class import to handle any custom spacing. 

```suggestion
import spacing from "@patternfly/react-styles/css/utilities/Spacing/spacing";
        className={spacing.mrSm}
```

Reply:
Sounds like what I was looking for, I'll check that thanks :+1: